### ISSUE TITLE
Survey button in Test Pilot door hanger reserves way too much space

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#530]
Found in http://testpilot-dev.us-east-1.elasticbeanstalk.com/__version__ (9bf2838358cfb7001ed54122406e9c9b9968e0c9)

``` json
{"version": "", 
  "source": "https://github.com/mozilla/testpilot", 
  "commit": "9bf2838358cfb7001ed54122406e9c9b9968e0c9"}
```

---
### Steps to reproduce:
1. Go to http://testpilot-dev.us-east-1.elasticbeanstalk.com/experiments and sign up/in.
2. Open the Test Pilot door hanger via the choppy icon in the toolbar.
### Actual results:

You can only see 5-6 words of the description at best, and the big green [Survey] button reserves a bunch of space which leads to some awkward looking truncation.

![firefox_test_pilot_-_universal_search_and_1__pdehaan_pdehaan_____zsh__and_automatically_add_vendor_prefixes_based_on_caniuse_by_openjck_ _pull_request__138_ _mozilla-services_push-dev-dashboard](https://cloud.githubusercontent.com/assets/557895/14027026/e0a00ef8-f1b2-11e5-966d-91ddc34961c4.png)

Full descriptions from the [/experiments](http://testpilot-dev.us-east-1.elasticbeanstalk.com/experiments) page (including where they get [truncated]) are:
- **Tab Cinema** &mdash; [A visual history of  your tabs in slides]how form
- **Universal Search** &mdash; [Search your bookmarks, history, and ]the web from a single place
- **Activity Stream** &mdash; [Find everything you've ever found, n]o matter what
### Expected results:

It may be nicer if we could show the full descriptions and make the survey button a bit smaller so it doesn't overlap the description (or just float the Survey button so it overlaps the description on hover, or whatever).

**Comment by ckprice** [comment]
Related: #503

**Comment by johngruen** [comment]
this should be resolved by #503

**Comment by johngruen** [comment]
Closing for #565
